### ISSUE TITLE
On hovering over the 'Copy to Clipboard' button tooltip is getting truncated.

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#704]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to any of the button and invoke it.
Now mouse hover on the 'Copy to clipboard' button and observe the issue
Actual Result:
On hovering over the 'Copy to Clipboard' button tooltip is getting truncated.

Note: 
Same issue is also observed for all 'Source Code'  & 'and for all 'Copy to clipboard' button present throughout the application.
Invoke 'Dialogs & flyouts' button >> Invoke 'Modal' button, same issue is observed for 'Modal Source Code' link  present under 'Documentation group
Same issue is observed for 'Expand/Collapse' button
Same issue is observed for all "Core Component" buttons available in entire application
Expected Result:
On hovering over the 'Copy to Clipboard' button tooltip should be visible clearly without any truncation.

User impact:
Users may not see the full tooltip, which can confuse them about the button's purpose.

MAS Reference Link: [MAS 1.4.13 – Content on Hover or Focus](https://aka.ms/MAS1.4.13)
 
Trap ID:[7.1 - Data Loss](https://aka.ms/Data_Loss) 

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
